Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.h

```diff
@@ -131,7 +131,7 @@ struct DataKey;
 }
 
 class NetworkConnectionToWebProcess final
-    : public RefCounted<NetworkConnectionToWebProcess>
+    : public ThreadSafeRefCounted<NetworkConnectionToWebProcess>
```

Comment:
Please use `ThreadSafeRefCounted<NetworkConnectionToWebProcess, WTF::DestructionThread::MainRunLoop>`. It is super dangerous otherwise. I don't think it'd be safe to destroy this on a non-main thread.

Reply:
Ah, great catch, I have addressed these issues in the latest PR.

Thanks for reviewing!